packages/spark-core/base/_inputs.scss

```diff
@@ -0,0 +1,105 @@
+// ==================================================================
+// Input Component
+// ==================================================================
+label,
+.sprk-b-Label {
+  @include TextCrop(
+    $line-height: $line-height-display-seven,
+    $top-crop: 8,
+    $bottom-crop: 8,
+    $crop-font-size: (strip-unit($font-size-display-seven) * 16),
+    $crop-line-height: $line-height-display-seven);
+
+  font-size: $font-size-display-seven;
+  color: $color-display-seven;
+  font-weight: $font-weight-display-seven;
+  line-height: $line-height-display-seven;
+}
+
+.sprk-b-Label--inline {
+  display: inline-block;
+}
+
+.sprk-b-SubLabel {
+  font-size: $font-size-body-two;
+  font-weight: $font-weight-body-two;
+}
+
+.sprk-b-TextInput,
+textarea {
+  color: $gray-dark;
+  outline: none;
+  font-size: $font-size-body-one;
+
+  &:focus {
+    border-color: $blue-medium;
+    box-shadow: 0 0 0 3px $blue-medium;
+  }
+}
+
+.sprk-b-TextInput--monetary {
+  position: relative;
+
+  &::before {
+    content: '$';
+    color: $gray-dark;
+    position: absolute;
+    top: 0.6rem;
+    left: 0.55rem;
+  }
+}
+
+.sprk-b-TextInput--date-picker {
+  background-image: url('../images/calendar.svg');
+  background-position: 5px 3px;
+  padding-left: 3rem;
+}
+
+.sprk-b-TextInput--error {
+  background-color: $red-light;
+  border-color: $red-medium;
+
+  &:focus {
+    border-color: $red-medium;
+    box-shadow: 0 0 0 3px $red-medium;
+    background-color: $red-light;
+  }
+}
+
+.sprk-b-TextInput[readonly] {
+  color: $white;
+
+  &:focus {
+    border-color: $gray-medium;
+    box-shadow: 0 0 0 3px $gray-medium;
+  }
+}
+
+.sprk-b-InputContainer > .sprk-b-Label,
+.sprk-b-InputContainer > label {
+  margin-bottom: $space-m;
+}
+
+.sprk-b-HelperText {
+  color: $gray-medium;
+}
+
+.sprk-b-ErrorText {
+  color: $red-medium;
+  font-size: $font-size-body-two;
```

Comment:
These should inherit the base-font-size, line height, weight from the `body` (scales settings) right? So, I don't think we need these declarations but I would double check.